minard/channeldb.py

```diff
@@ -359,12 +377,34 @@ def get_all_thresholds(run):
                         disc_average += int(threshold) 
                     else: 
                         count_max_thresholds += 1
-                        maxed_thresholds.append((crate,slot,channel))
                     online_channels += 1
 
     disc_average = round(float(disc_average) / online_channels, 3)
 
-    return disc, disc_average, count_max_thresholds, maxed_thresholds
+    return disc, disc_average, count_max_thresholds, message
+
+def get_maxed_thresholds(run):
+    """
+    Get the crate, card, channels of all maxed discriminator thresholds (255)
```

Comment:
I wonder if it's worth returning all thresholds >= 254 since we have sometimes bumped thresholds up and down by a single count.